vnets.tf

```diff
@@ -166,3 +166,40 @@ resource "azurerm_virtual_network_peering" "private_public" {
   allow_gateway_transit        = false
   use_remote_gateways          = false
 }
+
+##################################################################################################################
+###################   TRUSTED   ##################################################################################
+##################################################################################################################
+
+# Resources groups for TRUSTED NETWORK
+resource "azurerm_resource_group" "trusted" {
+  name     = "jenkinsinfra-trusted"
+  location = "East US"
+}
+
```

Comment:
```suggestion
```
(if the suggestion of including trusted in subnets inside the `private` vnet makes sense to you)